stages/org.osbuild.grub2

```diff
@@ -51,14 +51,18 @@ STAGE_OPTS = """
 
 def copy_modules(tree):
     """Copy all modules from the build image to /boot"""
+    source = "/usr/lib/grub/i386-pc"
+    if not os.path.exists(source):
```

Comment:
I think in general we should not make anything conditional on the tree or the buildtree. There could be more packages in the buildtree than we need (to reuse them across output types), meaning we get legacy stuff even on EFI images, or we could have failed to include the package we need and then we fail silently rather than complain loudly.

I would just make this an explicit option in the stage config.